packages/nextjs/test/integration/pages/unmatchedCustomPageExtension.someExtension

```diff
@@ -0,0 +1,3 @@
+This page simply exists to test the compatibility of Next.js' `pageExtensions` option with our auto wrapping
+process. This file should not be turned into a page by Next.js and our webpack loader also shouldn't process it.
+This page should not contain valid JavaScript.
```

Comment:
Can we test the negative here in some way?

Reply:
I figured that this already is testing the negative - not really by having a dedicated test, but rather by not crashing when such a file is provided.

We could theoretically try and open this page and wait for a 404 but at that point, I feel like we're testing Next.js itself. Did you have something else in mind?

Reply:
Yeah, I guess it would cause the build to fail if we tried to wrap it, wouldn't it? Okay, I think we're good here, thanks.